app-creator/src/widgets/draggable-widget/draggable-widget.ts

```diff
@@ -79,6 +80,37 @@ export class DraggableWidget extends LitElement {
   @property({ type: Boolean })
   editable = false;
 
+  /**
+   * Sets the editing widget's parent container border color to the default gray color.
+   */
+  static removeEditingWidgetHighlight() {
+    const editingWidget = store.getState().editingElement;
+
+    if (editingWidget == null) {
+      return;
+    }
+
+    const type = getIdPrefix(editingWidget.id) as WidgetType;
+
+    if (type === WidgetType.panel) {
+      const dropzone = editingWidget
+        .querySelector('dropzone-widget')
+        ?.shadowRoot?.querySelector(`#${CONTAINER_ID}`);
+
+      if (dropzone != null) {
+        (dropzone as HTMLElement).style.borderColor = 'var(--border-gray)';
```

Comment:
Instead of setting the styles directly, we should add this as a property on the dropzone-widget itself and let the dropzone figure out how to restyle its border. Same for setting the accent color below.